app/controllers/links_controller.rb

```diff
@@ -25,6 +25,8 @@ def external_url(type:, options: {})
       'https://projects.theforeman.org/projects/foreman/issues'
     when 'vmrc'
       'https://www.vmware.com/go/download-vmrc'
+    when 'docs'
+      docs_url(guide: options['section'], **options.slice('chapter', 'flavor'))
```

Comment:
I think generally you don't want to link to other flavors, but this is here so you can do something like "this requires Katello, see katello-flavor".